README.md

```diff
@@ -1,8 +1,13 @@
-# VIVIDUS Starter
+# VIVIDUS Training: Practice Part 1
 
-The purpose of this repository is to help users start creating automated tests using VIVIDUS.
+## Tests Execution and Analysis
 
-The instructions on how to setup and generate tests project can be found in the official ["Getting Started"](https://docs.vividus.dev/vividus/latest/getting-started.html) guide.
+1.	Clone this repo to your local machine & build the project as described in the Module 4
 
-## Still have questions?
-:postbox: [Contact Us](https://docs.vividus.dev/vividus/latest/index.html#_contract_us)
+2.	Import the project to your Eclipse workspace:
+    - Click File > Import > Existing Projects into Workspace
+    - Set the path to the built project and click "Finish" button
+
+3.	Run the tests located in /demo folder at your local machine using the default config
```

Comment:
how to run?

Reply:
Updated to "Execute tests located in /demo folder at your local machine as described in the Module 5 using the default config"